package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "vue-kanban",
-  "version": "1.6.2",
+  "name": "@blackdackel/vue-kanban",
```

Comment:
Can we revert this file and I will publish a new version when this has been merged into master.